VideoLocker/res/values/strings.xml

```diff
@@ -259,6 +259,7 @@
     <string name="msg_video_not_downloaded">"Unable to download video"</string>
     <!-- Error shown when video download fails -->
     <string name="msg_video_not_available">This video is currently unavailable.\nCheck back again later.</string>
+    <string name="video_only_on_web_label">This video is currently only on web. Click to view on web.</string>
```

Comment:
This should get a comment for translators explaining what the string is for, like all the other strings.

It's probably clear enough given that it's a full sentence, but having one for each string makes it more obvious that it's necessary and what needs to be done.

Let me know if you need help with that.